.github/workflows/merge-gate.yaml

```
@@ -255,6 +255,20 @@ jobs:
       runner: ${{ matrix.platform }}
       product: tt-nn
 
+  ttsim-unit-tests:
```

```suggestion
  ttsim-unit-tests:
    if: ${{ github.event_name != 'pull_request' }}
```

We want to run this:
- whenever there is a build (needs: does this)
- but not on pull requests because of external security issues (by filtering out pull_requests)

This line in `find-changes` job should cover this, but let's be defensive and explicit

```
  find-changes:
    if: github.event.action != 'destroyed' && github.event_name != 'pull_request'
```